app/facades/dashboard.rb

```diff
@@ -0,0 +1,7 @@
+class Dashboard
+  attr_accessor :user
```

Comment:
Probably `attr_reader` would be enough here.

Reply:
👍